repos/fdbt-site/src/utils/index.ts

```diff
@@ -241,3 +241,9 @@ export const convertDateFormat = (date: string): string => {
 };
 
 export const isReturnTicket = (ticket: Ticket | TicketWithIds): ticket is ReturnTicket => ticket.type === 'return';
+
+export const objectKeyMatchesExportNameExactly = (objectKey: string, exportName: string): boolean => {
+    const parts = objectKey.split('/');
+    const exportNamePart = parts[2];
```

Comment:
should we add a try catch and throw an error if parts[2] is undefined?

Reply:
Although I've just realised we need to be catering for objects in s3 that dont match this structure - so we cant throw an error.